petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -68,16 +68,12 @@ def test_primitive(self):
         self.assertEqual(ts.bin_col.dtype.type, np.object_, "Binary type column is not inferred correctly.")
 
     def test_delete(self):
-        test_path = "/tmp/petastorm_test"
-        Path(test_path).mkdir(parents=True, exist_ok=True)
-        Path(os.path.join(test_path, "dir1")).mkdir(parents=True, exist_ok=True)
-        with open(os.path.join(test_path, "file1"), "w") as f:
-            f.write("abc")
-        with open(os.path.join(test_path, "file2"), "w") as f:
-            f.write("123")
-        converter = SparkDatasetConverter(test_path, 0)
+        df = self.spark.createDataFrame([(1, 2), (4, 5)], ["col1", "col2"])
+        converter = make_spark_converter(df, 'file:///tmp/123')
+        local_path = converter.cache_file_path[7:]
+        self.assertTrue(os.path.exists(local_path))
         converter.delete()
-        self.assertFalse(os.path.exists(test_path))
+        self.assertFalse(os.path.exists(local_path))
```

Comment:
add a test for `standardize_path(...)`

replace all `xx[:7]` with `urlparse(xx).path`